resources/js/config.js

```diff
@@ -2,17 +2,22 @@
  * Config file of laravel nova visual composer package
  * You can override it with laravel publish command (documented in README.md)
  */
+
+// Register custom text size tool
+window.laraNovaVisualComposerSize = ['14px', '16px', '18px', '21px'];
+
 window.laraNovaVisualComposerConfig = {
   quill: {
     toolbarOptions: [
       ['bold', 'italic', 'underline', 'strike'],
-      [{'list': 'ordered'}, {'list': 'bullet'}],
-      [{'script': 'sub'}, {'script': 'super'}],
-      [{'indent': '-1'}, {'indent': '+1'}],
-      [{'header': [1, 2, 3, 4, 5, 6, false]}],
-      [{'color': []}, {'background': []}],
-      [{'font': []}],
-      [{'align': []}],
+      [{ list: 'ordered' }, { list: 'bullet' }],
+      [{ script: 'sub' }, { script: 'super' }],
+      [{ indent: '-1' }, { indent: '+1' }],
+      [{ header: [1, 2, 3, 4, 5, 6, false] }],
+      [{ color: [] }, { background: [] }],
+      [{ size: ['14px', '16px', '18px', '21px'] }],
```

Comment:
You can use same array of line 7 no ?

Reply:
Yep nice catch. I've also provided a default value just in case.
```
[{ size: window.laraNovaVisualComposerSize || ['14px', '16px', '18px', '21px'] }],
```